### PR TITLE
[docs] add new solver CATrustRegionMethod.jl

### PR DESCRIPTION
**Pull request by fadihamad94**
Adding a new solver to the documentation:
Add a new row for the CATrustRegionMethod solver to the table in `docs/src/installation.md`

### CODE REVIEW

**Comment by odow** [comment]
Hi @fadihamad94 (and, I see, @ohinder 🇳🇿), we have a checklist for this that needs to be completed before merging:

## Basic

 - [ ] Check that the solver is a registered Julia package
 - [ ] Check that the solver supports the long-term support release of Julia
 - [ ] Check that the solver has a MathOptInterface wrapper
 - [ ] Check that the tests call `MOI.Test.runtests`. Some test excludes are
       permissible, but the reason for skipping a particular test should be
       documented.
 - [ ] Check that the README and/or documentation provides an example of how to
       use the solver with JuMP

## Documentation

 - [ ] Add a new row to the table in `docs/src/installation.md`

## Optional

 - [ ] Add package metadata to `docs/packages.toml`

**Comment by fadihamad94** [comment]
HI @odow, below is the checklist:

## Basic

 - [ ] Check that the solver is a registered Julia package
 - [x] Check that the solver supports the long-term support release of Julia
 - [x] Check that the solver has a MathOptInterface wrapper
 - [x] Check that the tests call `MOI.Test.runtests`. Some test excludes are
       permissible, but the reason for skipping a particular test should be
       documented.
 - [x] Check that the README and/or documentation provides an example of how to
       use the solver with JuMP

## Documentation

 - [x] Add a new row to the table in `docs/src/installation.md`

## Optional

 - [ ] Add package metadata to `docs/packages.toml`

**Comment by fadihamad94** [comment]
Hi @odow , thank you for the comment. We updated the package name and registered it https://github.com/JuliaRegistries/General/pull/121222.

**Comment by odow** [comment]
Great! New packages take three days to merge in case someone wants to comment on the name, but it looks okay to me.

File: docs/src/installation.md [hunk 1]
```diff
@@ -162,7 +162,8 @@ The link in the `Solver` column is the corresponding Julia package.
 | [SeDuMi](http://sedumi.ie.lehigh.edu/)                                         | [SeDuMi.jl](https://github.com/jump-dev/SeDuMi.jl)                               | Manualᴹ | GPL      | LP, SOCP, SDP            |
 | [SHOT](https://github.com/coin-or/SHOT)                                        | [AmplNLWriter.jl](https://github.com/jump-dev/AmplNLWriter.jl)                   |        | EPL       | (MI)NLP                  |
 | [StatusSwitchingQP.jl](https://github.com/PharosAbad/StatusSwitchingQP.jl)     |                                                                                  |        | MIT       | LP, QP                   |
-| [Tulip.jl](https://github.com/ds4dm/Tulip.jl)                                  |                                                                                  |        | MPL-2     | LP                       |
+| [Tulip.jl](https://github.com/ds4dm/Tulip.jl)                                  |                                                                                  |        | MPL-2     | LP                       | 
+|[CAT.jl](https://github.com/fadihamad94/CAT-Journal)                           |   |     | MIT      | NLP |
```
Comment (odow) [inline docs/src/installation.md:166]:
```suggestion
|[CATrustRegionMethod.jl](https://github.com/fadihamad94/CATrustRegionMethod.jl)                           |   |     | MIT      | NLP |
```

**Comment by odow** [comment]
I guess I should update the checklist: can you add GitHub actions to test the package? 

You should also add TagBot: https://github.com/jump-dev/JuMP.jl/blob/master/.github/workflows/TagBot.yml

**Comment by fadihamad94** [comment]
@odow done. I added GitHub actions to test the package and added TagBot.

**Review by odow — APPROVED** [review]

**Pull request merged**